docs/security.md

```diff
@@ -103,7 +125,7 @@ sources (i.e. files and environment variables). For example
 ```
 will place the content of `spark/my-secret-file` into the `PLACEHOLDER` environment variable and the `target-secret-file` file
 as well as the content of `spark/my-secret-envvar` into the `SECRET_ENVVAR` and `placeholder-file`. In the case of binary
-secrets (tagged with `__dcos_base64__`, for example) the environment variable will still be empty because environment
+secrets, the environment variable will still be empty because environment
```

Comment:
Nit: I don't think this comma is required.

Reply:
The wording "environment variables cannot be assigned to binary values" seems in the wrong order (or the "to" should be removed).

Reply:
I will fix it.